app/uk/gov/hmrc/cdsimportsddsfrontend/domain/Classification.scala

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.cdsimportsddsfrontend.domain
+
+case class Classification(id: Option[String], identificationTypeCode: Option[String])
```

Comment:
i suspect that both members are required here - ie. you can't give a classification ID without a type - but i understand this hasn't been specified in the ticket.  but we can probably refine this when building the "real thing".

Reply:
Agree. Will model as DMS expects when develop the real app.